src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -50,7 +50,7 @@ const handler = async (req, res) => {
 export default withSentry(handler);
 ```
 
-You can include your DSN directly in these two files, or provide it in either of two environment variables, `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`.
+You can provide your DSN to `Sentry.init` by hardcoding it directly or using environment variables (such as `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`), in these two files.
```

Comment:
```suggestion
You can hardcode your DSN and other configuration values directly inside `Sentry.init` call, or use `process.env` placeholders, and Next.js built-in [environment variables](https://nextjs.org/docs/basic-features/environment-variables) support instead.
```

Reply:
I thought the idea was we were going to look for the env vars ourselves, in the SDK. No?

If that's the case, then I think we should tell people, basically, either hardcode it in those files or put it in the following env variables and we'll pick it up. Telling them to pick it up makes their lives no easier.

Reply:
Yes, I think we should do that. Just as we do here https://github.com/getsentry/sentry-javascript/blob/master/packages/gatsby/gatsby-node.js#L1-L15